syntax/internal/stdlib/stdlib.go

```diff
@@ -15,137 +16,188 @@ import (
 	"gopkg.in/yaml.v3"
 )
 
+// There identifiers are deprecated in favour of the namespaced ones.
+var DeprecatedIdentifiers = map[string]interface{}{
```

Comment:
it's used by the [vm pkg](https://github.com/grafana/alloy/pull/1516/files#diff-66a593dc06bce84c4effbc2c1cee642d893c78158e4900e04d9074d6f0dc7453R489) so it has to be exported or I could implement a getter. Between the two options, I prefer the exported var because it's less code + I don't see a big risk of it being modified by accident. But if you insist on the other one I can add getters for both Identifiers and DeprecatedIdentifiers for consistency

Reply:
Let's just leave it as it is for now.